homeassistant/components/monzo/config_flow.py

```diff
@@ -0,0 +1,86 @@
+"""Config flow for Monzo."""
+from __future__ import annotations
+
+from collections.abc import Mapping
+import logging
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_TOKEN
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers import config_entry_oauth2_flow
+
+from .const import DOMAIN
+
+
+class MonzoFlowHandler(
+    config_entry_oauth2_flow.AbstractOAuth2FlowHandler, domain=DOMAIN
+):
+    """Handle a config flow."""
+
+    DOMAIN = DOMAIN
+
+    reauth_entry: ConfigEntry | None = None
+    oauth_data: dict[str, Any]
```

Comment:
Please initialize this field in a constructor

Reply:
Sorry, I'm a bit confused here. There's nothing to init it with until it's set in `async_oayth_create_entry`

Reply:
I agree, the current code is preferred over initializing it.